src/Components/Navbar/NavbarItem/NavbarItem.tsx

```diff
@@ -9,7 +9,8 @@ export type NavbarItemType = {
   
 const NavbarItem: React.FC<NavbarItemType> = (props) => {
     return (
-        <Link 
+        <Link
+            data-testid={props.route}
```

Comment:
Why do we use `testid` here?

Reply:
We use it for element selecting in App.test.tsx. I had some problems with this test and I tried differnt ways of element selecting.